elliott/tests/test_find_builds_cli.py

```diff
@@ -1,41 +1,38 @@
-import asyncio
 import unittest
 from unittest import IsolatedAsyncioTestCase, mock
 from unittest.mock import AsyncMock, MagicMock
 
-from elliottlib import errata as erratalib
-from elliottlib.brew import Build
 from elliottlib.cli.find_builds_cli import _filter_out_attached_builds, _find_shipped_builds, find_builds_konflux
-from flexmock import flexmock
```

Comment:
mixing flexmock with unittest.mock wasn't playing well, especially w. AsyncMock, so replace it entirely